templates/includes/sidebar.html

```diff
@@ -1,6 +1,19 @@
-<ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
-</ul>
+<div class="sidebar">
+    {% if user.is_authenticated %}
+        <p>Welcome, 
+            <a href="{% url 'taxi:driver-detail' user.id %}">
```

Comment:
Ensure that `user.id` corresponds to a driver, as the current logic might not work as expected if the user model is not directly linked to the `Driver` model. Consider verifying the relationship between the user and driver models.